### PR TITLE
Other in staff photographer

### PR DESCRIPTION
**Pull request by jamesgorrie**
We've been told that people who are staff, but are not official photographers still submit photography, especially editors in the field.

From Robert H
> These should be treated as staff photography

other-photographer

### CODE REVIEW

File: kahuna/public/js/usage-rights/usage-rights-editor.html [hunk 1]
```diff
@@ -110,15 +110,29 @@
...
-                            ng:model="ctrl.model[property.name]"
-                            ng:required="property.required"
-                            ng:options="o for o in ctrl.getOptionsFor(property)">
-                            <option value="">None</option>
-                        </select>
+                        <div class="flex-container"
+                             ng:if="property.optionsMap"
+                             ng:init="otherValue = false">
+                            <select
+                                class="full-width input-"
```
Comment (akash1810) [inline kahuna/public/js/usage-rights/usage-rights-editor.html]: Extra `input-` class?

**Comment by theefer** [comment]
Simple enough :+1:

**Comment by akash1810** [comment]
:+1:

**Comment by theefer** [comment]
Random though (please ignore as far as this PR is concerned): later we could add a button to "It was me actually!!1" which would use the logged in user's name from their session.

**Pull request merged**